src/init.cpp

```diff
@@ -1404,7 +1404,8 @@ bool AppInitMain(const util::Ref& context, NodeContext& node, interfaces::BlockA
     node.chainman = &g_chainman;
     ChainstateManager& chainman = *Assert(node.chainman);
 
-    node.peerman.reset(new PeerManager(chainparams, *node.connman, node.banman.get(), *node.scheduler, chainman, *node.mempool));
+    assert(!node.peerman);
+    node.peerman = MakeUnique<PeerManager>(chainparams, *node.connman, node.banman.get(), *node.scheduler, chainman, *node.mempool);
```

Comment:
style nit in 10182aaa5bb2521ef6836d9984806968d3f46b24: When changing this, I'd prefer to use `std::make_unique`, not the deprecated wrapper.

Reply:
has this been addressed?

Reply:
Done